docs-source/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -66,9 +66,8 @@ The script will perform the following steps:
   in the registry (for example, Oracle Cloud Infrastructure Registry, Docker Hub, and such) as
   this image contains sensitive information about the domain including keys and
   credentials that are used to access external resources (for example, data source password).
-  For more information about
-  [domain home in image protection]({{<relref "/security/domain-security/image-protection.md#weblogic-domain-in-docker-image-protection">}}),
-  see the ***Security*** section.
+  For more information, see
+  [WebLogic domain in Docker image protection]({{<relref "/security/domain-security/image-protection.md#weblogic-domain-in-docker-image-protection">}}).
```

Comment:
This is a side note for @rosemarymarano :    We know that this relref syntax doesn't quite work - it'll redirect to the top of the file instead of to the specified # location.   You need to pretend there's an "_index.md" file in the directory even if there isn't, and do this instead: `/security/domain-security/image-protection/_index.md#weblogic-domain-in-docker-image-protection`.

Reply:
I saw a couple of others of these.  We should come up with a grep expression to suss them out and fix 'em.

Reply:
you can just remove the `.md` after `image-protection` and it works fine

Reply:
Okay, I'll test removing the ".md" and do the grep.